sbin/fireqos.in.in

```diff
@@ -421,10 +392,10 @@ service() {
 }
 
 fireqos_active_interfaces() {
-	$LS_CMD $FIREQOS_DIR/ 2>/dev/null |\
-		$GREP_CMD ".conf" |\
-		$TR_CMD "\n" " " |\
-		$SED_CMD -e "s/\.conf//g" -e "s/ \+/ /g"
+	@LS@ $FIREQOS_DIR/ 2>/dev/null |\
```

Comment:
Well - for the purpose of this pull I may not.

A full audit of quoting would go well beyond these changes and is what is probably required.